Dockerfile

```
@@ -1,11 +1,14 @@
-FROM --platform=$BUILDPLATFORM golang:1.20-bookworm AS build
+FROM golang:1.20-alpine AS build
```

```suggestion
FROM golang:1.20-alpine3.18 AS build
```
If you are linking libraries then specifying the minor alpine version matching the later stage may help prevent things from going wrong in the future if we temporarily drift from the latest version.

---

@andyasp good catch - I was just thinking about this fact in the meantime (my fault that we didn't specify Alpine version).